### PR TITLE
removed templates

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

File: openvino-requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 openvino==2022.1.0.dev20220302
-openvino-dev==2022.1.0.dev20220302
\ No newline at end of file
+openvino-dev==2022.1.0.dev20220302
+nncf@git+https://github.com/openvinotoolkit/nncf@464244204fc2c5e80c8164c17d8d266ccae50062#egg=nncf
+openmodelzoo-modelapi@ git+https://github.com/openvinotoolkit/open_model_zoo@ef556fee2cdd92488838b49ef8939c303992d89c#egg=openmodelzoo-modelapi&subdirectory=demos/common/python
```
Comment (evgeny-izutov) [inline openvino-requirements.txt:4]: Do we really need `openvino-requirements.txt` file if all OV-related code is moved outside?

Comment (Ilya-Krylov) [reply openvino-requirements.txt:4]: I can say that it won't be used in training_extensions, I don't mind to remove, as you wish.

Remove completely?

Comment (Ilya-Krylov) [reply openvino-requirements.txt:4]: nncf ? openmodelzoo ? I moved it from requirements/runtime.txt

**Comment by JihwanEom** [comment]
run ote-test

**Pull request merged**